internal/controllers/network/tests/import/00-import-network.yaml

```diff
@@ -11,13 +11,8 @@ spec:
   import:
     filter:
       name: import-external
-      adminStateUp: false
```

Comment:
Why were those removed?

Reply:
Because these fields are not available as filters.

https://github.com/k-orc/openstack-resource-controller/blob/f425289ccbdd05c154fa5ffdfd88e2a45888d5de/api/v1alpha1/network_types.go#L114-L129